app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -42,7 +43,7 @@ echo "#"
 if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     CONSUMER_RPM=$(mktemp --suffix .rpm)
 
-    curl --insecure --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
+    curl --insecure --output $CONSUMER_RPM <%= url %>/pub/katello-ca-consumer-latest.noarch.rpm
     yum localinstall $CONSUMER_RPM -y
     yum install subscription-manager -y
```

Comment:
I noticed a slight bug here. While the CONSUMER_RPM does require subscription-manager, if there is going to be an explicit install of sub-man it should happen *before* installing the consumer RPM to be safe. The consumer RPM reconfigures subscription-manager and we should take care not to override this accidentally.

Reply:
Given the dependency is there, does it make sense to explicitly install it? `yum localinstall` takes care of dependencies, right? So drop it?

Reply:
I will fix it in https://projects.theforeman.org/issues/30742, let's not mix it with changes in this PR.